src/client/components/MapHeader.tsx

```diff
@@ -285,7 +289,7 @@ const MapHeader = ({
         )}
         <Flex className="geolevel-button-group">{geoLevelOptions}</Flex>
       </Flex>
-      <Box sx={{ lineHeight: "1" }}>
+      <Box sx={{ lineHeight: "1", position: "fixed", right: "150px" }}>
```

Comment:
It's very rare that you want `position: "fixed";` - you almost always want  `position: "absolute";` instead (the exception that proves the rule is clarifying here: an element that is intended to remain fixed when the page scrolls _and_ be overlaid above normal page content, like a floating "Help" button, is a good candidate for using `position: "fixed";`).

`"fixed"` means absolute positioning in regards to the _viewport_, whereas `"absolute"` means absolute positioning in regards to the parent element (or parent's parent, etc.).

In particular for this case, this would mean that by using `"fixed"` positioning our styling would break if the header above this button changes height, whereas using `"absolute"` positioning it will not.

```suggestion
      <Box sx={{ lineHeight: "1", position: "absolute", right: "150px" }}>
```